openc3/lib/openc3/packets/packet.rb

```diff
@@ -1019,6 +1050,13 @@ def to_config(cmd_or_tlm)
       else
         config << "COMMAND #{@target_name.to_s.quote_if_necessary} #{@packet_name.to_s.quote_if_necessary} #{@default_endianness} \"#{@description}\"\n"
       end
+      if @accessor.class.to_s != 'OpenC3::BinaryAccessor'
+        config << "  ACCESSOR #{@accessor.class.to_s} #{@accessor.args.map { |a| a.to_s.quote_if_necessary }.join(" ")}\n"
+      end
+      # TODO: Add TEMPLATE_ENCODED so this can always be done inline regardless of content
+      if @template
+        config << "  TEMPLATE '#{@template}'"
+      end
```

Comment:
I don't understand this TODO comment

Reply:
I'll write the corresponding ticket.  In short we need a TEMPLATE keyword that takes base64 encoded data so a multiline string can be handled in one line of config file.

Reply:
https://github.com/OpenC3/cosmos/issues/1062